File: src/DashboardUI/src/data-contracts/OverlayDetails.ts
import { FeatureCollection } from 'geojson';

export interface OverlayDetails {
  waDEAreaReportingUuid: string;
  reportingAreaNativeID: string;
  waDEOverlayAreaType: string[];
  nativeReportingAreaType: string;
  state: string;
  areaLastUpdatedDate: string;
  organizationName: string;
  organizationState: string;
  organizationWebsite: string;
  geometry: FeatureCollection | null;
}

File: src/API/WesternStatesWater.WestDaat.Contracts.Client/OverlayDetails.cs

using GeoJSON.Text.Feature;

namespace WesternStatesWater.WestDaat.Contracts.Client;

public class OverlayDetails
{
    public string WaDEAreaReportingUuid { get; set; }
    public string ReportingAreaNativeID { get; set; }
    public List<string> WaDEOverlayAreaType { get; set; }
    public string NativeReportingAreaType { get; set; }
    public string State { get; set; }
    public DateTime? AreaLastUpdatedDate { get; set; }
    public string OrganizationName { get; set; }
    public string OrganizationState { get; set; }
    public string OrganizationWebsite { get; set; }
    public Feature Geometry { get; set; }
}